JuiceMaker/JuiceMaker/Controller/StockChangeViewController.swift

```diff
@@ -7,9 +7,51 @@
 
 import UIKit
 
-final class StockChangeViewController: UIViewController {
+protocol StockChangeDelegate: AnyObject {
+    func changeStock(with fruitInventory: [Fruit: Int])
+}
 
+final class StockChangeViewController: UIViewController {
+    @IBOutlet var fruitStockLabels: [UILabel] = []
+    @IBOutlet var stockSteppers: [UIStepper] = []
+    var fruitStore: FruitStore?
+    weak var delegate: StockChangeDelegate?
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        initializeComponents()
+        setStockStepperTag()
+    }
+    
+    @IBAction func updateFruitStock(_ sender: UIStepper) {
+        let index = sender.tag
+        let fruitStockLabel = fruitStockLabels[index]
+        let quantity = Int(sender.value)
+        guard let fruit = Fruit(rawValue: index) else { return }
+        
+        fruitStockLabel.text = "\(quantity)"
+        fruitStore?.updateStock(of: fruit, to: quantity)
+    }
+    
+    @IBAction func tapCloseButton(_ sender: UIBarButtonItem) {
+        guard let fruitStore = fruitStore else { return }
+        
+        delegate?.changeStock(with: fruitStore.fruitInventory)
+        dismiss(animated: true)
+    }
+    
+    private func initializeComponents() {
+        for (index, (fruitStockLabel, stockStepper)) in zip(fruitStockLabels, stockSteppers).enumerated() {
+            guard let fruit = Fruit(rawValue: index),
+                  let fruitCount = fruitStore?.getCurrentStock(of: fruit) else { return }
+            fruitStockLabel.text = "\(fruitCount)"
+            stockStepper.value = Double(fruitCount)
+        }
+    }
+    
+    private func setStockStepperTag() {
+        for (tag, stockStepper) in stockSteppers.enumerated() {
+            stockStepper.tag = tag
+        }
     }
```

Comment:
```suggestion
    private func initializeComponents() {
        for (index, (fruitStockLabel, stockStepper)) in zip(fruitStockLabels, stockSteppers).enumerated() {
            guard let fruit = Fruit(rawValue: index),
                  let fruitCount = fruitStore?.getCurrentStock(of: fruit) else { return }
            fruitStockLabel.text = "\(fruitCount)"
            stockStepper.value = Double(fruitCount)
            stockStepper.tag = index
        }
    }
```

초기 설정을 해주는 것이므로 `tag`를 할당하는 작업을 같이 수행할 수도 있을 것 같습니다.

Reply:
넵! 해당 내용 수정 완료했습니다 :) 감사합니다!